### DISCUSSION TITLE
option 'Send Email' also in System Configuration -> individual extension?

### DISCUSSION THREAD

**Discussion by Rolli1962** [akeeba/panopticon#309] [General]
Maybe it makes sense to allow also to set the option 'Send Email' for individual extensions in the general system configuration ?

I think it makes sense to let do updates automatically by panopticon, except known critical exceptions where only an email information should be send and the update has to be done manually - equal on which site.

Hope those would make sense in general?!

**Comment by nikosdion** [comment]
Dude. When An Extension Update Is Found -> Send Email. Individual extension -> Use global.

Also read #303 and #301.

**Reply by Rolli1962** [reply]
Have read it. But don't understand, why the option is not possible to set "Send Email" in System Configuration: Extension Name -> Update preference. There all options are possible except "Send Email".

**Reply by nikosdion** [reply]
Frankly, because I thought this makes no sense. If you really insist I can add it as an option.

**Reply by nikosdion** [reply]
I implemented your request. I still don't think it's a great idea, but since the scheduled reports features are not implemented yet it will make for a good stop-gap, I guess 😄
